.github/renovate.json

```diff
@@ -47,16 +47,7 @@
       ],
       "enabled": false,
       "matchPackageNames": [
-        "jakarta.servlet:jakarta.servlet-api"
-      ]
-    },
-    {
-      "description": "Contains incompatible API changes and needs compatibility work",
-      "matchManagers": [
-        "maven"
-      ],
-      "enabled": false,
-      "matchPackageNames": [
+        "jakarta.servlet:jakarta.servlet-api",
         "jakarta.servlet.jsp.jstl:jakarta.servlet.jsp.jstl-api"
```

Comment:
This exclusion remains valid, but not for the reason currently given. The exclusion remains valid for the reason given in the previous exclusion for `jakarta.servlet:jakarta.servlet-api`. Indeed, that reason applies to both `jakarta.servlet:jakarta.servlet-api` and `jakarta.servlet.jsp.jstl:jakarta.servlet.jsp.jstl-api`, so move these exclusions to the same section.